internal/backends/sqlite/metadata/registry.go

```diff
@@ -352,6 +354,27 @@ func (r *Registry) CollectionRename(ctx context.Context, dbName, oldCollectionNa
 	panic("not implemented")
 }
 
+// Version returns SQLite version.
+//
+// If there are no databases in the registry, empty string is returned.
+func (r *Registry) Version(ctx context.Context) (string, error) {
```

Comment:
> Registry provides access to SQLite databases and collections information.

SQLite library version is not a database or collection of information. It should not be there

Reply:
I don't think we should use another method at all. `state.Proivder.Get().HandlerVersion` is already there

Reply:
Ah, ok, I can try it. I misunderstood what you said yesterday - as it sounded to me that we should have `Info` interface for the backend that should return both name and version.

Reply:
That's indeed what I meant yesterday. But refactoring the telemetry reporter to accept the backend does not sound like a good idea, especially now. So, let's just™ store state the current backend in the state in addition to the version and use that.

Reply:
(and if we store the handler/backend in the state, let's remove it from `NewReporterOpts`)